core/domain/draft_upgrade_services_test.py

```diff
@@ -120,6 +144,98 @@ def test_convert_to_latest_schema_version_implemented(self):
+            'quot;abc2_height_120_width_120.png&amp;amp;amp;quot;\&amp;quot;'
+            '&amp;gt;&amp;lt;/oppia-noninteractive-image&amp;gt;&amp;lt;'
+            'p&amp;gt;You have opened the collapsible block.&amp;lt;/p'
+            '&amp;gt;&amp;quot;" heading-with-value="&amp;quot;Sample '
+            'Header&amp;quot;"></oppia-noninteractive-collapsible>')
+        draft_change_list = [
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'Intro',
+                'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+                'new_value': {
+                    'html': html_content,
+                    'content_id': 'content_id'
+                }
+            }),
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'Intro',
+                'property_name': (
+                    exp_domain.STATE_PROPERTY_INTERACTION_ANSWER_GROUPS),
+                'new_value': [{
+                    'rule_specs': [],
+                    'outcome': {
+                        'feedback': {
+                            'html': html_content,
+                            'content_id': 'cid'
+                        }
+                    },
+                    'training_data': [],
+                    'tagged_skill_misconception_id': None
+                }]
+            }),
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'Intro',
+                'property_name': exp_domain.STATE_PROPERTY_INTERACTION_HINTS,
+                'new_value': [{
+                    'hint_content': {
+                        'html': html_content,
+                        'content_id': 'content_id'
+                    }
+                }]
+            }),
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'Intro',
+                'property_name': (
+                    exp_domain.STATE_PROPERTY_INTERACTION_SOLUTION),
+                'new_value': {
+                    'answer_is_exclusive': True,
+                    'correct_answer': '',
+                    'explanation': {
+                        'content_id': 'cid',
+                        'html': html_content
+                    }
+                }
+            })]
+        converted_change_list = (
+            draft_upgrade_services.DraftUpgradeUtil._convert_states_v32_dict_to_v33_dict(  # pylint: disable=protected-access,line-too-long
+                'exp_id', draft_change_list))
+        self.assertEqual(
+            converted_change_list[0].to_dict()[u'new_value'][u'html'],
+            expected_output)
+        self.assertEqual(
+            converted_change_list[1].to_dict()[u'new_value'][0][u'outcome'][u'feedback'][u'html'], # pylint: disable=line-too-long
```

Comment:
You should not be using pylint-disable. Use intermediate variables if you need to. Also, I'm not sure you need the "u" prefixes here.